mlrun/package/packagers/python_standard_library_packagers.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import datetime
+import os
+import pathlib
+import tarfile
+import zipfile
+from enum import Enum
+
+from mlrun.artifacts import Artifact
+from mlrun.datastore import DataItem
+from mlrun.errors import MLRunInvalidArgumentError
+
+from ..common import ArtifactType
+from .default_packager import DefaultPackager
+
+
+# builtins packagers:
+class IntPackager(DefaultPackager):
+    PACKABLE_OBJECT_TYPE = int
+    DEFAULT_ARTIFACT_TYPE = ArtifactType.RESULT
+
+
+class FloatPackager(DefaultPackager):
+    PACKABLE_OBJECT_TYPE = float
+    DEFAULT_ARTIFACT_TYPE = ArtifactType.RESULT
+
+
+class BytesPackager(DefaultPackager):
+    PACKABLE_OBJECT_TYPE = bytes
+    DEFAULT_ARTIFACT_TYPE = ArtifactType.RESULT
+
+
+class StrPackager(DefaultPackager):
+    PACKABLE_OBJECT_TYPE = str
+    DEFAULT_ARTIFACT_TYPE = ArtifactType.RESULT
+
+    class ArchiveFormats(Enum):
+        ZIP = "zip"
+        TAR_GZ = "tar.gz"
+
+    @classmethod
+    def pack_file(cls, obj: str, key: str):
```

Comment:
IIUC it is like a best effort where the strpackager will try to resolve whether the returned string is a file and if so it will zip it and if not it will fall back to the defaultPackager and will just pickle it? or am I wrong? either way I think it would be nice to add some more description to this packager

Reply:
Basically the user can pass the artifact type to be 'file', 'directory', 'result' and 'object' for strings. But same as above: All of the packagers are yet to be implemented. The current ones are just for idea. I waited for full review before I proceed to implement them.